Running in wrapper mode for '219'

Running python job.

[name: "CPU-QUOTA"
int_val: 9
int_val: 63
int_val: 100
int_val: 94
int_val: 92
int_val: 36
int_val: 26
, name: "DISK"
int_val: 17
int_val: 13
int_val: 36
int_val: 5
int_val: 28
int_val: 21
int_val: 23
, name: "MEMORY"
int_val: 6
int_val: 12
int_val: 38
int_val: 15
int_val: 5
int_val: 36
int_val: 10
, name: "NET"
int_val: 30
int_val: 8
int_val: 6
int_val: 8
int_val: 32
int_val: 25
int_val: 14
]
The paramters are {u'CPU-QUOTA': array([  9,  63, 100,  94,  92,  36,  26]), u'NET': array([30,  8,  6,  8, 32, 25, 14]), u'DISK': array([17, 13, 36,  5, 28, 21, 23]), u'MEMORY': array([ 6, 12, 38, 15,  5, 36, 10])}
{'13.56.14.158': ['haproxy', 'node-app', 'mongo'], '18.144.43.31': ['node-app', 'mongo']}
Setting cpu quota to 157500 out of 250000
STRESSING VM_IP 13.56.14.158 AND CONTAINER d824175f3227, NET 40960.0
STRESSING VM_IP 13.56.14.158 AND CONTAINER d824175f3227, DISK 10905190.4
STRESSING VM_IP 13.56.14.158 AND CONTAINER d824175f3227, MEMORY 450000000.0
Recovering the memory utilization
Setting cpu quota to 22500 out of 250000
STRESSING VM_IP 13.56.14.158 AND CONTAINER cd40be619604, NET 153600.0
STRESSING VM_IP 13.56.14.158 AND CONTAINER cd40be619604, DISK 14260633.6
STRESSING VM_IP 13.56.14.158 AND CONTAINER cd40be619604, MEMORY 225000000.0
Recovering the memory utilization
Setting cpu quota to 250000 out of 250000
STRESSING VM_IP 13.56.14.158 AND CONTAINER c21ccad5695a, NET 30720.0
STRESSING VM_IP 13.56.14.158 AND CONTAINER c21ccad5695a, DISK 30198988.8
STRESSING VM_IP 13.56.14.158 AND CONTAINER c21ccad5695a, MEMORY 1425000000.0
Recovering the memory utilization
Setting cpu quota to 22500 out of 250000
STRESSING VM_IP 18.144.43.31 AND CONTAINER 01a0240fb06e, NET 153600.0
STRESSING VM_IP 18.144.43.31 AND CONTAINER 01a0240fb06e, DISK 14260633.6
STRESSING VM_IP 18.144.43.31 AND CONTAINER 01a0240fb06e, MEMORY 225000000.0
Recovering the memory utilization
Setting cpu quota to 250000 out of 250000
STRESSING VM_IP 18.144.43.31 AND CONTAINER d4df0b3c1712, NET 30720.0
STRESSING VM_IP 18.144.43.31 AND CONTAINER d4df0b3c1712, DISK 30198988.8
STRESSING VM_IP 18.144.43.31 AND CONTAINER d4df0b3c1712, MEMORY 1425000000.0
Recovering the memory utilization
Using 5 trials
INVALID EXPERIMENT TYPE: todo-app

----------------------------------------
Problem running the job:
(<type 'exceptions.SystemExit'>, SystemExit(None,), <traceback object at 0x7f4038474710>)
Traceback (most recent call last):
  File "/home/ubuntu/throttlebot/src/spearmint/spearmint/runner.py", line 42, in job_runner
    elif job.language == PYTHON: run_python_job(job)
  File "/home/ubuntu/throttlebot/src/spearmint/spearmint/runner.py", line 142, in run_python_job
    result = module.main(job.id, params)
  File "/home/ubuntu/throttlebot/src/spearmint/bayOptSearch/bayOpt.py", line 104, in main
    return explore_spearmint(workload_config, params)
  File "/home/ubuntu/throttlebot/src/spearmint/bayOptSearch/bayOpt.py", line 204, in explore_spearmint
    experiment_results = run_experiment.measure_runtime(workload_config, experiment_trials)
  File "/home/ubuntu/throttlebot/src/run_experiment.py", line 37, in measure_runtime
    exit()
  File "/usr/lib/python2.7/site.py", line 375, in __call__
    raise SystemExit(code)
SystemExit: None
None
----------------------------------------
Job file reloaded.
Could not find value in output file.
Job failed in 15.21 seconds.
